<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PGSale</class>
 <widget class="QWidget" name="PGSale">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidgetNor">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>1001</width>
     <height>207</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Detail</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableWidgetInd">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>380</y>
     <width>1001</width>
     <height>207</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Detail</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableWidgetCor">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>590</y>
     <width>1001</width>
     <height>207</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Detail</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="PBAdd">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>30</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QComboBox" name="selectBox">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>50</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Normal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Member</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineSearch">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>100</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="selectTypeBox">
   <property name="geometry">
    <rect>
     <x>761</x>
     <y>100</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Normal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Individual</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Corporate</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="PBRemove">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>90</y>
     <width>80</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PBSearch">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>100</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PBSave">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PBImport">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineVersion">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>选择一个版本</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PBRefresh">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>125</width>
     <height>17</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Promotion:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="discountLabel1">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>145</width>
     <height>17</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Points(per 50):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="discountLabel2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>30</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Level(per rank):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="discountLabel3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>198</width>
     <height>17</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>SPECIAL For Corporate:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="discountLabel4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>0</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; color:#ff0000;&quot;&gt;DISCOUNT!!!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-30</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:700;&quot;&gt;销售管理&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
